### ISSUE TITLE
When a user completes an activity, the game jumps too quickly to the next screen

### ISSUE THREAD

**Issue by chayaml** [ji-devs/ji-cloud#2308]
JiTap plays a congratulatory sound for a few seconds after a user completes an activity on a slide. With Jigzi, there is no sound, and the next page jumps too quickly; you do not have time to review what you've done.

**JiTap:**

https://user-images.githubusercontent.com/78186896/150244630-027829d9-03b2-4889-aba6-110a1db061b1.mov

https://jitap.net/activities/g9m3/play/

**Jigzi:**

https://user-images.githubusercontent.com/78186896/150243913-3cbac8a9-4904-491e-9d8c-537851741d45.mov

https://jigzi.org/admin/curation/717c6582-69d7-11ec-8549-c7faa0155ba1

**Comment by corinnewo** [comment]
Good point! I will ask @dakom what can be done with this. If it is a legacy player issue or a general player issue.

**Comment by dakom** [comment]
I think this is something which will be very common and should be on the jig player level. In other words, in addition to just `Next` there should be `NextWithEffect` with various effects like `NextWithEffect::Celebrate` or `NextWithEffect::FadeOut` or whatever. Once that exists, it's trivial to make the legacy module send those events instead of just `Next`

**Comment by corinnewo** [comment]
@MendyBerger I think we need to implement this - not for MVP though!
